setup.py

```diff
@@ -4,6 +4,8 @@
 from setuptools import setup, find_packages
 from setuptools.command.test import test as testcommand
 
+VERSION = __import__('pylmod').VERSION
```

Comment:
You should just use __version__ here instead of creating and referencing VERSION as a redundant module attribute